examples/dummy/glee.config.js

```diff
@@ -1,10 +1,15 @@
 // import customServer from './customServer.js'
 
 export default async function () {
-  // return {
-  //   websocket: {
-  //     httpServer: customServer,
-  //     adapter: 'socket.io', // Default. Can also be 'socket.io' or a reference to a custom adapter.
-  //   }
-  // }
+  return {
+    // websocket: {
+    //   httpServer: customServer,
+    //   adapter: 'socket.io', // Default. Can also be 'socket.io' or a reference to a custom adapter.
```

Comment:
Oh! Just caught an error that you didn't introduce but you may want to fix :)

```suggestion
    //   adapter: 'native', // Default. Can also be 'socket.io' or a reference to a custom adapter.
```